streams/src/main/java/org/apache/kafka/streams/state/StateSerdes.java

```diff
@@ -63,46 +73,105 @@ public StateSerdes(String stateName,
         this.valueSerde = valueSerde;
     }
 
+    /**
+     * Return the key serdes.
```

Comment:
Typo: "serdes" -> "serde" (singular, not plural).  Same for the similar lines below.